### DISCUSSION TITLE
STL下载网 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#157] [模板讨论]
STL下载网 模板问题反馈评论区
有问题可以下面反馈。

**Comment by schongux** [comment]
2023-05-15 03:47:44
日志：Failed at 1/4 request,
Fail assert: {"re": "快速登录", "from": "content"} from failed_asserts,
Request URL: https://www.stlxz.com/user/balance

**Reply by wjf0214** [reply]
cookie有效期就7天，过期了。
